.github/workflows/shared-app-deployment-workflow.yaml

```diff
@@ -0,0 +1,139 @@
+name: Shared Application Deployment Workflow
+
+on:
+  workflow_call:
+    secrets:
+      AZURE_TENANT_ID:
+        required: true
+        description: 'Azure Tenant ID'
+      AZURE_CLIENT_ID:
+        required: true
+        description: 'Azure Client ID'
+      AZURE_SUBSCRIPTION_ID:
+        required: true
+        description: 'Azure Subscription ID'
+      ACTIONS_AUTH_APP_ID:
+        required: true
+        description: 'GitHub App ID'
+      ACTIONS_AUTH_APP_PRIVATE_KEY:
+        required: true
+        description: 'GitHub App Private Key'
+    inputs:
+      environment:
+        type: string
+        required: true
+        description: 'Environment'
+      applicationFolder:
+        type: string
+        required: true
+        description: 'Application Folder'
+      applicationName:
+        type: string
+        required: true
+        description: 'Application Name'
+      applicationVersion:
+        type: string
+        required: true
+        description: 'Application Version'
+
+jobs:
+  deploy_app:
+    environment: ${{ inputs.environment }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        regionShortCode: ['ne', 'we']
+    steps:
+      - name: checkout repo
+        uses: actions/checkout@v1
+
+      - name: gh-app-install token
+        id: gh-app
+        uses: getsentry/action-github-app-token@v2
+        with:
+          app_id: ${{ secrets.ACTIONS_AUTH_APP_ID }}
+          private_key: ${{ secrets.ACTIONS_AUTH_APP_PRIVATE_KEY }}
+
+      - name: checkout devops-azure repo
+        uses: actions/checkout@v3
+        with:
+          path: devops-azure
+          repository: Uniphar/devops-azure
+          token: ${{ steps.gh-app.outputs.token }}
+
+      - name: azure login
+        uses: azure/login@v1
+        with:
+          client-id: ${{ secrets.AZURE_CLIENT_ID }}
+          tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+          subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+          enable-AzPSSession: true
+
+      - name: compute environment variables
```

Comment:
is there a particular reason to use (maybe somewhere else, but I only noticed now) this extension instead of a regular powershell bash?? 

We mentioned before that this extension has some issues and doesn't fail when it should fail.

Not sure if you'll have an extra overhead of having to sign in... though.. I think we replaced "most" of its usage on devops-azure